### PR TITLE
OP-1130 improve JPA methods - disease package

### PR DESCRIPTION
**Pull request by dbmalkovsky**
See OP-1130

Changes for disease package

Matches changes in CORE PR https://github.com/informatici/openhospital-core/pull/1121

### CODE REVIEW

File: src/main/java/org/isf/disease/rest/DiseaseController.java [hunk 1]
```diff
@@ -271,9 +272,10 @@ public ResponseEntity<DiseaseDTO> updateDisease(@Valid @RequestBody DiseaseDTO d
...
-		if (diseaseManager.updateDisease(disease) != null) {
-        	return ResponseEntity.ok(diseaseDTO);
-        } else {
+		try {
+			diseaseManager.updateDisease(disease);
+			return ResponseEntity.ok(diseaseDTO);
+		} catch (OHServiceException serviceException) {
         	throw new OHAPIException(new OHExceptionMessage("Disease not updated."), HttpStatus.INTERNAL_SERVER_ERROR);
         }
 	}
```
Comment (mwithi) [inline src/main/java/org/isf/disease/rest/DiseaseController.java:281]: indentation problem here

File: src/main/java/org/isf/disease/rest/DiseaseController.java
Comment (mwithi) [inline src/main/java/org/isf/disease/rest/DiseaseController.java:1]: line 314 and 315 also seems have indentation problem

File: src/main/java/org/isf/disease/rest/DiseaseController.java (hunk 1 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/disease/rest/DiseaseController.java]: @mwithi Fixed it; thanks

File: src/main/java/org/isf/disease/rest/DiseaseController.java
Comment (mwithi) [reply src/main/java/org/isf/disease/rest/DiseaseController.java:1]: actually from 310 to 315

Comment (dbmalkovsky) [reply src/main/java/org/isf/disease/rest/DiseaseController.java:1]: fixed I think

Comment (mwithi) [reply src/main/java/org/isf/disease/rest/DiseaseController.java:1]: yes! thanks!

**Review by mwithi — APPROVED** [review]
LGTM. Thanks a lot!

**Pull request merged**